samples/subsys/usb/dfu/dts.overlay

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
You don't need this anymore, you only need to enable `CONFIG_BOOTLOADER_MCUBOOT` in the `.conf`

Reply:
I think I can remove it completely, it is only needed if it is part of MCUboot based bootloader.

Reply:
You need `CONFIG_BOOTLOADER_MCUBOOT` if you are building an application that is to be booted by MCUboot, so any app using this DFU mechanism.

Reply:
You are right